pkgs/servers/ser2net/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, gensio, libyaml }:
+
+stdenv.mkDerivation rec {
+  pname = "ser2net";
+  version = "4.1.1";
+
+  src = fetchurl {
+    url = "https://downloads.sourceforge.net/sourceforge/ser2net/${pname}-${version}.tar.gz";
```

Comment:
Consider using github repository: https://github.com/cminyard/gensio

Reply:
Done, thanks for the review.